mapzen-places-api/src/test/java/com/mapzen/places/api/internal/PlaceAutocompletePresenterTest.java

```diff
@@ -49,6 +55,31 @@ public void getBoundingBox_shouldReturnCorrectBox() throws Exception {
   }
 
   @Test
+  public void getBoundingBox_lostClient_shouldReturnCorrectBox() throws Exception {
+    FusedLocationProviderService.FusedLocationProviderBinder stubBinder =
+          mock(FusedLocationProviderService.FusedLocationProviderBinder.class);
+    FusedLocationProviderService mockService = mock(FusedLocationProviderService.class);
+      when(stubBinder.getService()).thenReturn(mockService);
+    FusedLocationProviderApiImpl impl =
+        (FusedLocationProviderApiImpl) LocationServices.FusedLocationApi;
+    impl.onServiceConnected(stubBinder);
```

Comment:
Wow this does require a bit of mocking to get around the fact that `LocationServices.FusedLocationApi` is declared `static final` doesn't it? Just brainstorming here but I wonder if we can build some sort of testing utils into `Lost` to make it easier to unit test against?